src/main/kotlin/app/batch/MessageUtils.kt

```diff
@@ -0,0 +1,74 @@
+package app.batch
+
+import com.beust.klaxon.JsonObject
+import com.beust.klaxon.Parser
+import com.beust.klaxon.lookup
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import java.nio.ByteBuffer
+import java.text.SimpleDateFormat
+import java.util.zip.CRC32
+
+@Component
+open class MessageUtils {
```

Comment:
If this is a direct import from HTME then maybe we should add it to a shared library too? It might sit well alongside the docker images for HBase too which I mentioned in another comment

Reply:
Again - we'll do the shared libraries "soon" - though, please do raise a story for those and get it prioritised with Nick.